packages/shared/src/const.ts

```diff
@@ -30,6 +30,12 @@ export const Env = {
   /** Number of hours to assume a role for, @default 8 */
   AwsRoleDurationHours: 'AWS_ROLE_DURATION_HOURS',
 
+  /** AWS role config bucket */
+  AwsRoleConfigBucket: 'ROLE_CONFIG_BUCKET',
```

Comment:
@blacha  Do you know where to set the values for these environments variables?

Reply:
It should be inside of `_infra` where the lambda is deployed